<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConservacionForm</class>
 <widget class="QGroupBox" name="ConservacionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>151</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Conservaci√≥n</string>
  </property>
  <property name="title">
   <string>CONSERVACION</string>
  </property>
  <widget class="QLineEdit" name="cuentaEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>261</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="consUsoLbl">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>104</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Uso original:</string>
      </property>
      <property name="buddy">
       <cstring>consUsoEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="consUsoEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="consReconvertidaLbl">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>104</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Reconvertida:</string>
      </property>
      <property name="buddy">
       <cstring>consReconvertidaEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="consReconvertidaEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="consObraMarchaLbl">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>104</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Tipo de Obra:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="consObraCmb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Ninguno</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="consEstadoLbl">
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>104</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Estado:</string>
      </property>
      <property name="buddy">
       <cstring>consEstadoCmb</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="consEstadoCmb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Ninguno</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
